templates/partials/committee-report-house-senate.html

```diff
@@ -0,0 +1,46 @@
+{% with committee=context() %}
+
+{% set table_data = [
+     (reports.0.individual_itemized_contributions_period, 'Itemized Individual Contributions'),
+     (reports.0.individual_unitemized_contributions_period, 'Unitemized Individual Contributions'),
+     (reports.0.total_individual_contributions_period, 'Total Individual Contributions'),
+     (reports.0.political_party_committee_contributions_period,'Party Committee Contributions'),
+     (reports.0.other_political_committee_contributions_period,'Other Committee Contributions'),
+     (reports.0.candidate_contribution_period, 'Candidate Contributions'),
+     (reports.0.total_contributions_period, 'Total Contributions'),
```

Comment:
Oh sorry. Yeah I think we should leave them out becuase there's currently
no way of showing them as a cumulative total of the other lines.

On Wed, May 13, 2015 at 4:20 PM, Lindsay Young <notifications@github.com>
wrote:

> In templates/partials/committee-report-house-senate.html
> <https://github.com/18F/openFEC-web-app/pull/158#discussion_r30285849>:
>
> > @@ -0,0 +1,46 @@
> > +{% with committee=context() %}
> > +
> > +{% set table_data = [
> > +     (reports.0.individual_itemized_contributions_period, 'Itemized Individual Contributions'),
> > +     (reports.0.individual_unitemized_contributions_period, 'Unitemized Individual Contributions'),
> > +     (reports.0.total_individual_contributions_period, 'Total Individual Contributions'),
> > +     (reports.0.political_party_committee_contributions_period,'Party Committee Contributions'),
> > +     (reports.0.other_political_committee_contributions_period,'Other Committee Contributions'),
> > +     (reports.0.candidate_contribution_period, 'Candidate Contributions'),
> > +     (reports.0.total_contributions_period, 'Total Contributions'),
>
> We went back and forth on it, I thought we ended it up saying we were just
> going to show everything. We can take them out if you want.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/openFEC-web-app/pull/158/files#r30285849>.
>



-- 
Noah Manger
18F <http://18f.gsa.gov> | Work: 415-696-6146